backend/dal/Services/PropertyService.cs

```diff
@@ -110,7 +110,9 @@ public Property Get(int id)
                 .ThenInclude(a => a.Province)
                 .Include(p => p.Address)
                 .ThenInclude(a => a.Country)
-                .Where(p => p.Id == id)
+                .Include(p => p.Leases)
+                .ThenInclude(l => l.Tenant)
+                .Where(p => p.Id == 153)
```

Comment:
Did you want to hardcode the `.Where(...)`?

Reply:
oops, this was just for testing purposes, will remove.